requirements.txt

```diff
@@ -1,5 +0,0 @@
-pyquil>=3.0.0,<4.0.0
```

Comment:
Why was this file deleted?

Reply:
It was unused outside of CI, wasn't mentioned in the `README`, and had the potential to get out of sync with the requirements hardcoded in `setup.py`. I've since added it back, per my note above, but am using it in `setup.py` to ensure that there's a single source of truth on dependencies.

Reply:
I agree! 💯